src/sdk/account/toNexusAccount.ts

```diff
@@ -113,6 +113,9 @@ export type NexusSmartAccountImplementation = SmartAccountImplementation<
     encodeExecute: (call: Call) => Promise<Hex>
     encodeExecuteBatch: (calls: readonly Call[]) => Promise<Hex>
     getUserOpHash: (userOp: Partial<UserOperationStruct>) => Promise<Hex>
+    setActiveValidationModule: (validationModule: BaseValidationModule) => void
+    getActiveValidationModule: () => BaseValidationModule
+    getSigner: () => Promise<Signer>
```

Comment:
Let's not expose this unless there's a good reason? It's kind of a duplicate because you can access the signer stuff the via: `nexusAccount.client` (it's the masterClient).